platformio.ini

```diff
@@ -47,8 +47,9 @@ default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, nodemcuv2_160, esp8266_2m_1
 src_dir  = ./wled00
 data_dir = ./wled00/data
 build_cache_dir = ~/.buildcache
```

Comment:
I'll pull this into my branch, it's mostly unreviewed
I suspect we should just contribute this part upstream to get a good review on it
and so we don't have to continuously merge